account_invoice_partner_no_autofollow/models/account_move.py

```diff
@@ -0,0 +1,29 @@
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def message_subscribe(self, partner_ids=None, channel_ids=None, subtype_ids=None):
+        if self.env.context.get("no_auto_follow"):
+            return False
```

Comment:
The goal of the module is to prevent only the invoiced partner from being subscribed.
Here you block **all** partners from subscription.

Reply:
fixed